src/store/root.store.ts

```diff
@@ -1,5 +1,8 @@
 import { rootReducer } from './root.reducer';
 import { saga } from './root.saga';
 import { createStore, applyMiddleware } from 'redux';
+import { watchFetchPets } from './result/sagas/result.saga';
 
 export const store = createStore(rootReducer, applyMiddleware(saga));
+
+saga.run(watchFetchPets);
```

Comment:
Move your saga to the `root.saga.ts` as other developers did. Please contact @IvanBonk or @okhabr for more information

Reply:
Okay, understood: )